src/components/Datepicker/Datepicker.vue

```diff
@@ -438,23 +442,30 @@ export default {
       if (this.internalValue === 'Invalid Date') this.internalValue = ''
     },
 
-    isDateDisabled() {
-      let valid = false
-      if (this.disabledPast && dayjs().isAfter(this.internalValue, 'day')) {
-        valid = true
-      } else if (
+    isDateDisabledPast() {
+      return this.disabledPast && dayjs().isAfter(this.internalValue, 'day')
+    },
+
+    isMinDateDisabled() {
+      return (
         this.minDate &&
         dayjs(this.internalValue).isSameOrBefore(this.minDate, 'day')
-      ) {
-        valid = true
-      } else if (
+      )
+    },
+
+    isMaxDateDisabled() {
+      return (
         this.maxDate &&
         dayjs(this.internalValue).isSameOrAfter(this.maxDate, 'day')
-      ) {
-        valid = true
-      }
+      )
+    },
 
-      return valid
+    isDateDisabled() {
+      return !!(
+        this.isDateDisabledPast() ||
```

Comment:
Should this become _computed properties_ since they don't get any params?

Reply:
Good suggestion, I will check.